pkg/varlinkapi/images.go

```diff
@@ -413,17 +413,26 @@ func (i *LibpodAPI) DeleteUnusedImages(call iopodman.VarlinkCall) error {
 }
 
 // Commit ...
-func (i *LibpodAPI) Commit(call iopodman.VarlinkCall, name, imageName string, changes []string, author, message string, pause bool) error {
+func (i *LibpodAPI) Commit(call iopodman.VarlinkCall, name, imageName string, changes []string, author, message string, pause bool, format string) error {
 	ctr, err := i.Runtime.LookupContainer(name)
 	if err != nil {
 		return call.ReplyContainerNotFound(name)
 	}
 	sc := image.GetSystemContext(i.Runtime.GetConfig().SignaturePolicyPath, "", false)
+	var mimeType string
+	switch format {
+	case "oci":
+		mimeType = buildah.OCIv1ImageManifest
+	case "docker":
+		mimeType = manifest.DockerV2Schema2MediaType
+	default:
+		return call.ReplyErrorOccurred(fmt.Sprintf("unrecognized image format %q", format))
```

Comment:
I would prefer to set the default to OCI instead of returning an error.  This is similar to what we do in the command line.

Reply:
Default here would be an unrecognized string - that definitely sounds like an error to me?

Reply:
Ok, fair enough ... @QiWang19 leave as is